data/json/monsters/insect_spider.json

```diff
@@ -401,62 +401,179 @@
   {
     "id": "mon_centipede_small",
     "type": "MONSTER",
-    "name": { "str": "huge centipede" },
-    "description": "A brightly colored, many-legged mutant centipede the size of a dog.",
+    "name": { "str": "centipede" },
+    "description": "A brightly colored mutant centipede the size of a small dog, skittering quickly in the underbrush.  Probably won't think you're suitable prey, but beware its venom.",
+    "symbol": "a",
+    "color": "light_green",
     "copy-from": "mon_centipede_giant",
-    "proportional": { "hp": 0.5, "speed": 1.1, "morale": 0.67, "melee_dice_sides": 0.67 },
-    "volume": "30 L",
-    "weight": "40 kg",
-    "upgrades": { "half_life": 14, "into": "mon_centipede_giant" }
+    "volume": "15 L",
+    "weight": "20 kg",
+    "hp": 30,
+    "speed": 115,
+    "attack_cost": 115,
+    "aggression": 0,
+    "morale": 0,
+    "melee_skill": 5,
+    "melee_dice_sides": 3,
+    "special_attacks": [
+      {
+        "type": "bite",
+        "cooldown": 10,
+        "move_cost": 115,
+        "damage_max_instance": [ { "damage_type": "stab", "amount": 2, "armor_penetration": 8 } ],
+        "min_mul": 3,
+        "max_mul": 5.4,
+        "//": "Bite does 6-10 dmg, armor penetration 5, 8 hits to max venom intensity",
+        "effects": [ { "id": "venom_dmg", "duration": 250, "affect_hit_bp": true } ]
+      }
+    ],
+    "dodge": 4,
+    "armor_bash": 1,
+    "armor_cut": 3,
+    "armor_bullet": 2,
+    "armor_stab": 2,
+    "anger_triggers": [ "PLAYER_CLOSE" ],
+    "fear_triggers": [ "HURT" ],
+    "upgrades": { "age_grow": 30, "into_group": "GROUP_CENTIPEDE_ADULT" }
   },
   {
     "id": "mon_centipede_giant",
     "type": "MONSTER",
     "name": { "str": "giant centipede" },
-    "description": "A meter-long centipede with a menacing pair of pincers, moving swiftly on dozens of spindly legs.",
+    "description": "A two meter long centipede with a menacing set of venom claws and a worrying number of grabbing appendages, moving swiftly on dozens of spindly legs.  It has a pair of small compound eyes, but seems to mostly orient itself with its antennae.",
```

Comment:
```suggestion
    "description": "A two meter long centipede with a menacing set of venom-dripping claws and a worrying number of grabbing appendages, moving swiftly on dozens of spindly legs.  It has a pair of small compound eyes, but seems to mostly orient itself with its antennae.",
```
Or just "venomous".

Reply:
The English name for them is poison/venom claws, and the Latin name for them is obscure enough that I had to look it up. It's not optimal, but to me venom-dripping/venomous claws would imply that they were claws on the ends of their legs.

(Alternately I could just stop being such a pedant and go with mandibles, since that's a more commonly known term)

Reply:
I fear that "venom claws" already implies they're on the legs, at least to a non-entomologist like myself. "mouth-claws", perhaps?

Reply:
Decided to suck it up and go with mandibles. I can only hope all the other entomonerds in the audience can forgive me.